pkg/cluster/k8sres.go

```diff
@@ -290,6 +290,26 @@ func nodeAffinity(nodeReadinessLabel map[string]string) *v1.Affinity {
 	}
 }
 
+func generatePodAffinity(labels labels.Set, topologyKey string, nodeAffinity *v1.Affinity) *v1.Affinity {
+	// generate pod anti affinity to avoid multiple on instances on the same node
```

Comment:
pls change to sth like

> generate pod anti-affinity to avoid multiple pods of the same Postgres cluster on the same node

to make it a little bit more clear

Reply:
I've changed the comment as requested and also added documentation to admin docs and operator params reference.

CRD-based operator configuration is missing the parameters as I wasn't able to incorporate the changes.

Reply:
Implemented propagation of the pod anti affinity parameters to/from CRD-based operator configuration.